homeassistant/components/incomfort/climate.py

```diff
@@ -32,7 +29,7 @@ def __init__(self, client, room):
         self._room = room
         self._name = 'Room {}'.format(room.room_no)
 
-    async def async_added_to_hass(self):
+    def async_added_to_hass(self) -> None:
```

Comment:
This should be a coroutine.

Reply:
Sorry, this was a WIP - I hadn't expected it to be reviewed!